pom.xml

```diff
@@ -685,21 +685,72 @@
                             <goal>prepare-agent</goal>
                         </goals>
                         <configuration>
-                            <append>true</append>
+                            <destFile>${project.build.directory}/jacoco-output/jacoco-unit-tests.exec</destFile>
```

Comment:
"append" was removed, was this deliberate? What's the effect of this?

Reply:
Yes, Now we have a merged report with both unit test and integration just like whois-internal.

Reply:
Ah, it was already done in whois-internal, thanks